egs/babel/s5d/run-1-main-unicode.sh

```diff
@@ -96,6 +97,7 @@ if [[ ! -f data/lang/L.fst || data/lang/L.fst -ot $lexicon ]]; then
     data/local $oovSymbol data/local/tmp.lang data/lang
 fi
 
+exit
```

Comment:
Yeah. Good catch. I was testing the lexicon portion so I added an exit that I forgot to get rid of.